canonical-web-and-design/ubuntu.com
# Grid updates - the final chapter

## Done

- updated grid patterns on remaining u.c pages according to point 2 of the [Vanilla upgrade guide](https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide).

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the following pages to ensure that grid/row/col layouts don't appear broken, ignoring other issues caused by the Vanilla upgrade (such as broken stepped lists, spacing on secondary navigation, p-card header icon misalignment):
  - [x] /public-cloud
  - [x] /resources
  - [x] /rfp
  - [x] /server/contact-us
  - [x] /server/hyperscale
  - [x] /server/thank-you
  - [x] /server/maas/thank-you
  - [x] /support
  - [x] /support/community-support
  - [x] /support/thank-you
  - [x] /takeovers
  - [x] /telecommunications
  - [x] /training
  - [x] /whitepapers/beta/robotics
  - [x] /whitepapers/robotics


## Issue / Card

Fixes #5688
